api/TimeAddressableMediaStore.yaml

```diff
@@ -1019,6 +1043,8 @@ paths:
       responses:
         "200":
           $ref: '#/components/responses/trait_resource_listing_head_200'
+        "404":
+          description: The requested flow does not exist.
```

Comment:
Should this be a 404 if the flow ID path component is invalid and not when the Flow does not exist? If a Flow does not exist it returns `[]`.